template/cypress/support/commands.ts

```diff
@@ -0,0 +1,56 @@
+import { User } from '@prisma/client';
+
+import '@testing-library/cypress/add-commands';
+import { LOGIN_TOKEN_KEY } from '../../constants';
+import { LoginTaskObject } from '../plugins';
+import { UserAttrs } from '../../tests/factories/user';
+
+declare global {
+  // eslint-disable-next-line
+  namespace Cypress {
+    interface Chainable {
+      login: typeof login;
+    }
+  }
+}
+
+// -- This is a parent command --
```

Comment:
Are we keeping this here for information or should it be removed?

Reply:
it's generated by default when you install Cypress. We can definitely take it out if you think it's not helpful.